extensions/exchanges/bitstamp/exchange.js

```diff
@@ -0,0 +1,326 @@
+var Bitstamp = require('bitstamp')
+  , path = require('path')
+  , Pusher = require('pusher-js/node')
+  , colors = require('colors')
+
+var args = process.argv
+
+var wsOpts = {
+  encrypted: true,
+  pairOk: false,
+  currencyPair: 'btcusd',
+  trades: {evType: 'trade', channel: 'live_trades'},
+  quotes: {evType: 'data', channel: 'order_book'}
+}
+
+// The use of bitstamp-ws  requires that
+// Knowledge of the asset/currency pair
+// before the first call for a trade
+// As zenbot dont returns the currency pair
+// before the first trade is requested
+// it has been neccessary to get it from
+// t:he command line arguments
+args.forEach(function(value) {
```

Comment:
I dont quite understand the meaning of this. The internal logic is still a bit "secret". At least I have added comments where this implementation differs from the others. Any suggestions are welcome from a node programmer. This is project is my first attempt to live in the "callback hell".